ui/pages/LoginPage.qml

```diff
@@ -33,7 +35,14 @@ SimpleLayout {
 
             text: qsTr("SSO")
 
-            onClicked: Auth.grant()
+            onClicked:
+            {
+
+                var conf = GetOAuth2Config
```

Comment:
This is asynchrone. You need to wait GetOAuth2Config to receive a dataResponse before doing Auth.updateConfig and Auth.grant